apps/api-extractor/src/cli/RunAction.ts

```diff
@@ -52,11 +55,45 @@ export class RunAction extends CommandLineAction {
         + ' normally be performed for a ship/production build. For example, the *.api.ts'
         + ' review file is automatically copied in a local build.'
     });
+
+    this._typescriptLibPackagePath = this.defineStringParameter({
+      parameterLongName: '--typescript-lib-package',
```

Comment:
>'--typescript-lib-package' [](start = 25, length = 26)

Maybe call this `--typescript-compiler-folder`?  Nobody cares which compiler library is used by API Extractor.  The important idea is that API Extractor needs to know which compiler version should be used to analyze the source files.